src/tools/python/templates/OpenTrackIO.html

```diff
@@ -92,35 +92,122 @@ <h4>Multicast Addressing</h4>
+                large and complex networks.
+            </p>
+            
+            <h4>Packet Header</h4>
+            <p>When using UDP transport, each packet should include the header below:</p>
+            <p>
+                <table>
+                  <thead>
+                    <tr>
+                      <th>Bit Offset</th>
+                      <th>Field</th>
+                      <th>Description</th>
+                    </tr>
+                  </thead>
+                  <tbody>
+                    <tr>
+                      <td>0-31</td>
+                      <td>Identifier</td>
+                      <td>4 bytes: Static value to indicate OpenTrackIO packet, set to ASCII "OTrk" (<code>0x4F54726B</code>)</td>
+                    </tr>
+                    <tr>
+                      <td>32-39</td>
+                      <td>Reserved</td>
+                      <td>1 byte: This field is reserved for future use and should be ignored by both Producers and Consumers.</td>
+                    </tr>
+                    <tr>
+                      <td>40-47</td>
+                      <td>Encoding</td>
+                      <td>1 byte: Indicates the payload format (e.g., JSON = <code>0x01</code>, CBOR = <code>0x02</code>, OTP = <code>0x02</code>). <code>0x80</code> and above are reserved for vendor specific protocols.</td>
+                    </tr>
+                    <tr>
+                      <td>48-63</td>
+                      <td>Sequence number</td>
+                      <td>1 bytes: Unsigned integer indicating the OpenTrackIO packet's unique sequence number (<code>0x01</code> to <code>UINT16</code>)</td>
+                    </tr>
+                    <tr>
+                      <td>64–95</td>
+                      <td>Segment offset</td>
+                      <td>4 bytes: A 32-bit field indicating the byte offset of this payload segment when the overall payload length necessitates segmentation. Must be set to <code>0x00</code> for single-segment payloads.</td>
+                    </tr>
+                    <tr>
+                      <td>96</td>
+                      <td>Last segment flag</td>
+                      <td>This bit shall be set to <code>1</code> if this is the only segment or the last segment in a segmented payload, or<code>0</code> if more segments are expected.</td>
+                    </tr>
+                    <tr>
+                      <td>97–111</td>
+                      <td>Payload Length</td>
+                      <td>15 bits: Total length of the payload for the current packet (in bytes).</td>
+                    </tr>
+                    <tr>
+                      <td>112–127</td>
+                      <td>Checksum (Fletcher16)</td>
+                      <td>2 bytes: A 16-bit checksum computed using the Fletcher-16 algorithm, covering the header and payload (excluding checksum bytes)</td>
```

Comment:
Don't we need to say that you should zero out the checksum when generating the packet?